install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+
+[ ! -d ~/.tldr ] && printf "tldr seems to have already been previously installed.\nDelete the \'$HOME/.tldr\' directory first before relaunching this script\n" && exit 1 || mkdir -p ~/.tldr/cache
+
+download() {
+  hash curl 2>/dev/null && curl -sf -o "$1" "$2" || { hash wget 2>/dev/null && wget -qO "$1" "$2"; } || echo "Fail to download $2" && exit 1
```

Comment:
I think it's better to check if the `curl` or `wget` is present to avoid the user to install the one or the other package. Often `wget` is already present by default, but if `curl` is present, better to use it because it's faster (for me at least) and I also prefer it, but this is only my personal opinion.